run_tests.sh

```diff
@@ -0,0 +1,4 @@
+python3.7 -m unittest discover && \
```

Comment:
I was using `pytest` which can do automatic test discovery including doctest discovery. Do we want to specifically document some testing tool we can use to make test running simpler?

Reply:
(Note: Similar, runner tools like tox, etc.)

Reply:
You should have said something : P
The reason I converted the tests was because when I was adding things to the parser, I needed better assertion failure messages than are provided by plain assert. 
If we want to use pytest as a test runner, then yeah, we should add a requirements.txt and document it.

Reply:
Also, I can change the doctest command to: `python3 -m doctest -v *.py`